src/countryflag/countryflag.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import countryflag from 'countryflag'
+
+class CountryFlag extends Component {
+  render() {
+    const { iso2, svg } = this.props
+    const flag = countryflag(iso2)
+    return svg === true || flag.emoji === null ? (
+      <img style={{ width: '18px' }} alt={flag.name} src={flag.svg} />
```

Comment:
Can we change `width: '18px'` for `width: 1em` so the image takes the size of the line where it's included?

Reply:
and why not move the size to a `defaultProp` value ? that way we could customise from outside easily.